cdc/owner/changefeed.go

```diff
@@ -811,6 +811,18 @@ func (c *changefeed) releaseResources(ctx context.Context) {
 	c.initialized.Store(false)
 	c.isReleased = true
 
+OUT:
```

Comment:
Please add comments to explain why we should drain the warnings before releaseResoruces @sdojjy